src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -553,8 +553,15 @@ private void AttributePrefix(IEnumerable<HtmlSymbol> whitespace, IEnumerable<Htm
             }
             else
             {
+                // Output the attribute name, the equals and optional quote. Ex: foo="
+                Output(SpanKind.Markup);
```

Comment:
Exactly why is this addition required?  `SkipToAndParseCode()` often calls `Output()` itself.